mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
+  
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
 
```

Comment:
In the [predicates homework](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2/tree/main/exercises/D-predicates), you would have seen that in programming, things that start with a verb such as `is`, `should`, `can` usually indicate it should return a boolean. 

I think you should put your expression on line 23, so that you don't need to change line 25.

Reply:
@louisechow  I wrote it this way now, let me know if it is the right way: 

```
function greaterThan10(num) {
  let isBigEnough = num > 10;

  if (isBigEnough) {
    return "num is greater than 10";
  } else {
    return "num is not big enough";
  }
}
```